src/test/java/core/basesyntax/CalculationTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.BalanceTransaction;
+import core.basesyntax.strategy.PurchaseTransaction;
+import core.basesyntax.strategy.SupplyTransaction;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CalculationTest {
```

Comment:
Actually, you don't have any class named Calculation. So, let's rename it to FruitCalculatorImplTest and test the methods of this particular class